src/models/exhaust-csv-export.ts

```diff
@@ -0,0 +1,54 @@
+import * as path from 'path';
+import * as fs from 'fs/promises';
+import {ERRORS} from '../util/errors';
+import {ExhaustPluginInterface} from './exhaust-plugin';
+const {InputValidationError} = ERRORS;
+
+export class ExhaustCsvExporter implements ExhaustPluginInterface {
```

Comment:
we should use functional approach, let's go through tomorrow

Reply:
I know, it's a stub (that's why the PR is default)